src/selectors/entities/channels.js

```diff
@@ -521,7 +522,9 @@ export const getSortedPublicChannelWithUnreadsIds = createIdsSelector(
             const channel = channels[id];
             return !favoriteIds.includes(id) &&
                 teamChannelIds.includes(id) && channel.type === General.OPEN_CHANNEL;
-        }).map((id) => channels[id]).sort(sortChannelsByDisplayName.bind(null, locale));
+        }).map((id) => channels[id]).
+        sort(sortChannelsByDisplayName.bind(null, locale)).
+        sort(sortMutedChannels.bind(null, myMembers));
```

Comment:
I'm not sure here about the concatenation of the sorts. Accordingly to the [documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort) the sort implementation in Javascript is not necessarily ["stable"](https://en.wikipedia.org/wiki/Sorting_algorithm#Stability) . I tested it manually in the last version of Firefox, Chromium and Edge, and everything looks like correct (but I'm not sure about older versions). For me is enough, but up to you to combine them in an single sort function if you think is needed.